discord/cog.py

```diff
@@ -41,6 +41,7 @@
     Type,
     TypeVar,
 )
+from asyncio import create_task
```

Comment:
I think it's better to just import the `asyncio` module instead of directly importing one thing from the `asyncio` module.